zulipterminal/ui_tools/views.py

```diff
@@ -1179,7 +1180,10 @@ def __init__(self, controller: Any, msg: Message, title: str,
         self.time_mentions = time_mentions
 
         msg_info = [
-            ('', [('Date & Time', time.ctime(msg['timestamp'])[:-5]),
+            ('', [('Date & Time', datetime.datetime.fromtimestamp(
```

Comment:
This shouldn't need an extra module to import. The `time.ctime` method provides a full string containing the year too.

On a side note, it is good to mention any added module to the dependency files (Pipfile, requirements, setup), if the module isn't listed there :)

Reply:
`datetime` is part of the Python standard library just like `time`. I don't think we need to mention it in the dependency files.

This is what `ctime` gives us, `Wed Jan 26 07:06:55 2021` whereas `datetime` allows us to call `strftime` which let's us format it however we want... `Jan 20 2021, 07:06:55` . 

Latter one is better as Year at the end doesn't feel intuitive. (atleast to me)

Also in the code I noticed a  `TODO` mentioning that maybe in the future we may add 12-hour format support with application wide support for different formats.
 `datetime` is the perfect module for that.

If you want I can change it back to `time.ctime` OR as `time.ctime` is used only two times in this file, I can change the second instance to use `datetime` and then `time` module import can be removed?

Reply:
Ah, yes. Thanks for correcting. `datetime` doesn't need mentioning. (You may take that comment for a module not in standard python library :sweat_smile:)

Good catch. I didn't take that TODO into consideration. Now I get your point.

You may update the PR accordingly :+1: (mentioning the `time` to `datetime` in commit text)

I merely dabble iin reviewing currently. Apologies for that :sweat_smile:

Reply:
Re dependencies: as you've agreed, this isn't an issue.

Note that the message format has the year at the left, to allow it to look aligned when the year is optional, so we use an adjusted form of `ctime` already, in some sense.

There is an open PR I've not got to yet which approaches the 12/24-hour issue, so let's look at that for comparison. Given the dependency of formatting on user setting, it may make sense to place a standard formatter in the model, moved from the MessageBox.

Reply:
@neiljp should I change the code back to use ctime?